src/extension.ts

```diff
@@ -178,3 +178,18 @@ export function activate(this: any, context: ExtensionContext) {
 }
 
 export function deactivate() {}
+
+/**
+ * Checks for the explicit setting of the IS_DEVELOPEMENT_MODE and
+ * Implcitly checks by using the magic session string. This session value is used whenever an extension
+ * is running on a development host. https://github.com/microsoft/vscode/issues/10272
+ */
+function getTelemetry() {
+  if (process.env.IS_DEVELOPEMENT_MODE === 'true' || env.sessionId === 'someValue.sessionId') {
```

Comment:
This checks if it's "true", do we want to check if it's "development" instead? Also nit, `IS_DEVELOPEMENT_MODE` has an extra `E` in it.

Reply:
Good catches. Typo on my part as I was switching things around!